frontend/components/Players/DraggableQueue.vue

```diff
@@ -41,7 +41,9 @@
             <like-button :item="item" />
           </v-list-item-action>
           <v-list-item-action v-if="!isPlaying(index)" class="mr-2">
-            <item-menu :item="item" />
+            <v-btn icon @click="removeFromQueue(item)">
+              <v-icon>mdi-delete-outline</v-icon>
```

Comment:
I mention this already the review's main message, but I think ``mdi-playlist-minus`` fits better.

Reply:
I do agree with the choice, the trash can seems very out of place, however, I didn't really find a good alternative before. The icon is addressed in https://github.com/jellyfin/jellyfin-vue/pull/1753/commits/16249a688a65bad8bf7d6afa60609867d6114c06.